### PR TITLE
fix(ios): Give a time delay to scroll if search view is hiding on selection

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-26519

 -User action must. So no test case.

### CODE REVIEW

File: iphone/Classes/TiUITableView.h [hunk 1]
```diff
@@ -84,6 +83,8 @@
 }
 
 @property (nonatomic, assign) BOOL viewWillDetach;
+@property (nonatomic, assign) BOOL isLoaded;
```
Comment (janvennemann) [inline iphone/Classes/TiUITableView.h]: Can you be more specific here with the variable name? What exactly is `isLoaded` associated with?

File: iphone/Classes/TiUITableViewProxy.m [hunk 2]
```diff
@@ -168,6 +168,26 @@ - (NSIndexPath *)indexPathFromInt:(NSInteger)index
...
+        __block NSInteger rowPosition = 0;
+        for (int i = 0; i < section; i++) {
+          searchedRow = searchIndex[i];
+          rowPosition += searchedRow.count;
+        }
+
+        searchedRow = searchIndex[section];
+
+        [searchedRow enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *_Nonnull stop) {
+          if (idx == row)
```
Comment (janvennemann) [inline iphone/Classes/TiUITableViewProxy.m]: Please use braces even for single line statements.

**Review by janvennemann — CHANGES_REQUESTED** [review]

**Review by janvennemann — APPROVED** [review]

**Comment by janvennemann** [comment]
@vijaysingh-axway CR passed! Can this even be unit tested? Looks like it requires user interaction.

**Review by ssjsamir — APPROVED** [review]
FR Passed: Tableview search no longer throws an error when going to a specific position. Tested with the following test case:
```
var win = Ti.UI.createWindow({
backgroundColor : '#fff'
});
 
var self = Ti.UI.createView({
borderRadius : 10,
backgroundColor : 'red',
});
 
win.add(self);
 
var label1 = Ti.UI.createLabel({
text : 'Este es el Titulo',
color : "#333",
font : {
fontSize : 12
}
});
 
var toolbar = Ti.UI.createToolbar({
items : [label1],
top : 0,
});
 
var tableData = [];
for (var i = 0; i <= 50; i++) {
var row = Ti.UI.createTableViewRow({
id_colu : i,
title : 'columna' + i,
height : 40
});
 
var title = Ti.UI.createLabel({
text : 'otracolu' + i,
});
 
row.add(title);
tableData.push(row);
}
 
var searchBar = Ti.UI.createSearchBar({
barColor : '#000',
showCancel : true,
height : 43,
top : 5,
filterAttribute : 'title'
});
 
var table = Ti.UI.createTableView({
data : tableData,
objName : 'table',
search : searchBar,
filterAttribute : 'title',
scrollable : true,
top : 90,
bottom : 10,
});
 
table.addEventListener('click', function(e) {
table.scrollToIndex(e.index, {
animated : false,
position : Ti.UI.iOS.TableViewScrollPosition.TOP
});
});
 
win.add(toolbar);
self.add(table);
 
win.open();
```

**Test Environment**
```
iPhone 6S+ sim(12.1)
APPC CLI: 7.0.10
Operating System Name: Mac OS Mojave
Operating System Version: 10.14.2
Node.js Version: 8.9.1
Xcode 10.1
```

**Comment by lokeshchdhry** [comment]
@vijaysingh-axway , Can you please add unit test for this.

**Pull request merged**